src/guides/v2.3/extension-dev-guide/intro/developers_roadmap.md

```diff
@@ -12,15 +12,92 @@ To satisfy the minimum required elements for creating or customizing your Magent
 
 *  [Declare component dependencies]({{ page.baseurl }}/extension-dev-guide/build/composer-integration.html) in `composer.json`.
```

Comment:
If we extend this info with the example, we may clarify where exactly the dependencies are declared.

```suggestion
*  [Declare component dependencies]({{ page.baseurl }}/extension-dev-guide/build/composer-integration.html) in the `require` and/or `require-dev` section of the `composer.json`.
```